scripts/add_electricity.py

```diff
@@ -419,12 +419,20 @@ def attach_wind_and_solar(
             else:
                 capital_cost = costs.at[car, "capital_cost"]
 
+            if not ppl.query("carrier == @car").empty:
+                caps = ppl.query("carrier == @car").groupby("bus").p_nom.sum()
+                caps = pd.Series(data = caps, index = ds.indexes["bus"]).fillna(0)
+            else:
+                caps = pd.Series(index = ds.indexes["bus"]).fillna(0)
+
```

Comment:
What do these changes have to do with keeping existing caps in the sector model?

Reply:
Ah no! These are for adding renewables from custom_powerplants.csv. Sorry, I thought I had mentioned that in the commit message

Reply:
Ah, I see. Ok! I guess we can keep it in the same PR, it's not too different. Thanks!